### ISSUE TITLE
When connecting an additional screen, all panels move there

### ISSUE THREAD

**Issue by 7Z0t99** [mate-desktop/mate-panel#434]
My setup is as follows: I've got a PC that is connected to a monitor and a TV. The monitor is used for work and the TV when watching movies, photos, etc. 
The problem is that when one works at the monitor and somebody else switches on the TV to watch regular TV channels, the panels get moved to the TV, where one can't see them.
I expect the panels to stay on their screen when a new screen is plugged in / switched on. Ideally this should be the default, but I'm also interested to hear about ways to configure this.

**Comment by seamus-45** [comment]
Setting "Primary" monitor in display setup does not work?

**Comment by raveit65** [comment]
As always if openning reports here please provide general info's :unamused: 
Distro?
Mate version?
package version?

**Comment by clefebvre** [comment]
Hi,

I can't reproduce this issue. I think it's to do with the fact that you didn't configure it.

Each "set" of monitor has a memorized configuration. So, to configure what happens when both the TV and the monitor are plugged, you need to have both plugged and launch the Display setup tool.

In that tool, you select the monitor and click on "Set as Primary" and then click "Apply".

**Comment by 7Z0t99** [comment]
I'm on debian testing, which currently has mate-panel 1.12.2-1.

I didn't know that MATE memorizes the configuration. Setting Primary helps. 
My problem with this approach is that you can't reach the monitor settings from an empty desktop <- that's what you see on the first screen when you plug-in a new monitor for the first time. On windows, you just right-click on the desktop to go to the monitor settings. Which program does display the context menu on the desktop: is it also mate-panel?

The current way panels are configured regarding multiple monitor is just not optimal, see e.g. also #438

**Comment by seamus-45** [comment]
>  Which program does display the context menu on the desktop: is it also mate-panel?
No. It is caja. However, you can add the shortcut with command `mate-display-properties`, and bind it on Win+P

**Comment by 7Z0t99** [comment]
Well, I expected there was a way to create a keyboard shortcut to open the settings and I tried. It works, even though MATE calls the combination Mod4+P instead of Super+P or Win+P.
I was surprised that before, Win+P didn't do anything. Can't we make Win+P opening mate-display-properties the default?

**Comment by 7Z0t99** [comment]
Oh and please, can we get a better name for the key "Mod4"?

**Comment by janw-oostendorp** [comment]
I have the same issue. I also just confirmed this in a clean (and updated) VM
[My issue](https://github.com/mate-desktop/mate-panel/issues/438)

Is there a Mate pannel API? Then maybe I could manipulate this by mself.

**Comment by raveit65** [comment]
Does this issue still exists?
